pkgs/by-name/tl/tlmi-auth/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, meson
+, ninja
+, openssl
+, cmake
+}:
+let
+  name = "tlmi-auth";
+  version = "1.0.1";
+in
+stdenv.mkDerivation {
+  pname = name;
+  version = version;
+
+  src = fetchFromGitHub {
+    owner = "lenovo";
+    repo = name;
+    rev = "v${version}";
+    hash = "sha256-/juXQrb3MsQ6FxmrAa7E1f0vIMu1397tZ1pzLfr56M4=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    ninja
+    pkg-config
+    cmake
```

Comment:
Why do we need cmake here? It doesn't look like tlmi-auth uses it.

Reply:
Nice catch! it's not used at all. `cmake` is now removed.